tools/crm_mon_print.c

```diff
@@ -699,6 +699,11 @@ print_failed_actions(pcmk__output_t *out, pe_working_set_t *data_set,
     return rc;
 }
 
+#define CHECK_RC(rc, val)    \
+    if (val == pcmk_rc_ok) { \
+        rc = pcmk_rc_ok;     \
+    }
```

Comment:
It's a good idea to name macro arguments something reasonably unique ... otherwise you get a situation like in another commit:

            CHECK_RC(rc, out->message(out, "failed-fencing-history", hp, unames,
                                      is_set(mon_ops, mon_op_fence_full_history),
                                      rc == pcmk_rc_ok));

The "rc == pcmk_rc_ok" in that could be ambiguous in macro expansion. In this case it happens to be the desired variable either way, but it's easy to imagine a situation where a variable named rc is used in val but is not used as the macro's rc argument. I say easy because it's bit me before ;-)

Reply:
Okay, I think I've made this much better.  You'll definitely want to double check.